src/components/Sidebar/Sidebar.tsx

```diff
@@ -1,43 +1,68 @@
-import React from "react";
-import { Box, Flex, Button } from "@radix-ui/themes";
+import React, { useState } from "react";
+import { Box, Flex, Button, IconButton } from "@radix-ui/themes";
+import { BarChartIcon } from "@radix-ui/react-icons";
 import styles from "./sidebar.module.css";
 import { ChatHistory, type ChatHistoryProps } from "../ChatHistory";
 import { Settings } from "./Settings";
+import { Statistic } from "../Statistic/Statistic";
+import { useEventBusForChat } from "../../hooks";
 
 export const Sidebar: React.FC<
   {
     onCreateNewChat: () => void;
   } & ChatHistoryProps
 > = ({ history, onHistoryItemClick, onCreateNewChat, onDeleteHistoryItem }) => {
+  const [isOpenedStatistic, setIsOpenedStatistic] = useState(false);
+  const { backFromChat } = useEventBusForChat();
+
+  const handleCloseStatistic = () => {
+    setIsOpenedStatistic(false);
+  };
   return (
     <Box className={styles.sidebar}>
-      <Flex
-        direction="column"
-        position="fixed"
-        left="0"
-        bottom="0"
-        top="0"
-        style={{
-          width: "inherit",
-        }}
-      >
-        <Flex mt="4" mb="4">
-          <Button
-            variant="outline"
-            ml="auto"
-            mr="auto"
-            onClick={onCreateNewChat}
-          >
-            Start a new chat
-          </Button>
-        </Flex>
-        <ChatHistory
-          history={history}
-          onHistoryItemClick={onHistoryItemClick}
-          onDeleteHistoryItem={onDeleteHistoryItem}
+      {isOpenedStatistic ? (
+        <Statistic
+          onCloseStatistic={handleCloseStatistic}
+          backFromChat={backFromChat}
```

Comment:
yeap, I also think that may be it is better to create own event bus for statistic. I am going to do it today.

I've connected rendered statistic with vscode, but pr for vscode isn't created yet because I don't have permission yet